### PR TITLE
Improve README instructions 

### PR DESCRIPTION
**Pull request by Toxiapo**
This PR updates the README decryption section on how to install the app with the private package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+<img alt="Docker Image Version (latest semver)" src="https://img.shields.io/docker/v/nypl/patron-web?label=Docker%20Hub&logo=docker&sort=semver">
 </a>
-<img alt="GitHub Workflow Status" src="[https://img.shields.io/github/workflow/status/nypl-simplified/circulation-patron-web/CI?label=Tests&logo=github](https://img.shields.io/github/workflow/status/nypl-simplified/circulation-patron-web/CI?label=Tests&logo=github)">
-<img alt="GitHub Workflow Status" src="[https://img.shields.io/github/workflow/status/nypl-simplified/circulation-patron-web/Production](https://img.shields.io/github/workflow/status/nypl-simplified/circulation-patron-web/Production) Release?label=Build%20%28master%29&logo=github">
-<img alt="GitHub Workflow Status" src="[https://img.shields.io/github/workflow/status/nypl-simplified/circulation-patron-web/Publish](https://img.shields.io/github/workflow/status/nypl-simplified/circulation-patron-web/Publish) beta?label=Build%20%28beta%29&logo=github">
```
Comment (Toxiapo) [inline README.md:9]: I don't see the publish-beta in the workflows so I removed this.

**Review by samanthaandrews — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -160,10 +159,12 @@ The application will start at the base URL of `localhost:3000`. (NOTE: `npm run
 
 This app supports read online for encrypted books only in the AxisNow format, and if you have access to the [Decryptor](https://github.com/nypl-simplified-packages/axisnow-access-control-web)
 
+You will have to have a valid access token from npm to install the decryptor package, and set it to the environment variable `GH_PACKAGE_AUTH_TOKEN`
+
 To run with decryption:
 
-- Run `npm login --registry=https://npm.pkg.github.com`. You will need a Github Personal Access Token to use as your password.
-- Run `npm install` as normal.
+- Request an access token from [npm](https://docs.npmjs.com/creating-and-viewing-access-tokens#creating-access-tokens)
```
Comment (kristojorg) [inline README.md:166]: I think this is meant to be a github personal access token no? Since it's published on Github package registry and not npm registry

Comment (Toxiapo) [reply README.md:166]: Oh yes, that's correct. Thank you!

**Review by kristojorg — APPROVED** [review]
looks good! thanks

**Pull request merged**